### ISSUE TITLE
Parse changelog for "What's new" section

### ISSUE THREAD

**Issue by szhajdu** [arvai/trierhu#53]
Check some libraries:
- JS parser: https://www.google.de/?gfe_rd=cr&ei=ch0VWOv3Gc7j8wf4qYWgCQ#safe=off&q=markdown+parser+in+javascript
- PHP parser: https://www.google.de/?gfe_rd=cr&ei=ch0VWOv3Gc7j8wf4qYWgCQ#safe=off&q=markdown+parser+in+php

**Comment by arvai** [comment]
Good idea , but I think changelog should contain also the deep techincal stuff, but 'Whatsnew' is for people. So this 2 shouldn't be the same, in my opinion.

**Comment by szhajdu** [comment]
It depends on what do you want to show in What's new section. It is almost the same like the changelog and I don't think that we need deep technical stuff. Anyway these are just suggestions. The choice is yours :)
